docs/metecho-how-to.md

```diff
@@ -0,0 +1,219 @@
+- Contact an admin for the GitHub repository you want to contribute to (e.g.
+  Nonprofit Success Pack) on GitHub and ensure that you and anyone you’d like to
+  work with are collaborators.
+
+If you want to create a project, skip down to [Create a
+Project](#create-a-project).
+
+## Step 1 - Select a Project
+**Project** - *(e.g. Nonprofit Success Pack) Projects contain all the work being
+done. They are equivalent to GitHub repositories.*
+
+- Select the Project you want to contribute to.
+- If you do not see the Project you want to work on:
+  - Confirm that you are logged into the correct Metecho account.
+  - Contact an admin for the repository on GitHub and ensure that you are a
+    collaborator.
+  - Select Re-sync Projects in Metecho.
+
+To continue, go to **Step 2 - Create a Task** or, if you have a set of several
+closely related Tasks in mind, skip to [Step 13 - Create an
+Epic](#step-13---create-an-epic)
+
+## Step 2 - Create a Task
+**Task** - *Tasks represent small changes to the Project, and may stand alone or
+be part of an Epic. Creating a Task creates a branch in GitHub.*
+
+- Select the Tasks tab.
+- Create a new Task of your own (or from an existing GitHub Issue).
+  - Name the Task.
+  - The name can be a brief description of the work.
+  - Use the optional description section for more detail, if needed.
+  - Use the recommended dev org type unless you know of a specific reason to use
+    a different one.
+  - Select Create to save and navigate to your Task detail view. Select Create &
+    New to continue creating multiple Tasks that are unrelated to each other.
+- If you need to edit the Task name or description, or delete the Task, select
+  the gear icon in the top right corner. Deleting a Task deletes all the Orgs as
+  well.
+
+## Step 3 - Assign Task Developer
+**Developer** - *The person assigned to do the work of a Task.*
+
+- Assign yourself or another person to be the Developer for each Task you
+  created.
+- If you do not see the person you want to assign, contact an admin for the
+  repository on GitHub and ensure that the person is a collaborator. Then
+  re-sync GitHub collaborators in Metecho.
+- If you need to change or remove the Task Developer, use the drop down menu on
+  the Developer card in the Task detail view.
+
+## Step 4 - Create Dev Org
+**Dev Org** - *A temporary Salesforce org where a Developer can work on
+contributions to a Project. Dev Orgs expire after 30 days, and all unretrieved
+work is deleted.*
+
+- Review the Next Steps for This Task list to see progress and who is
+  responsible for the next step.
+- To begin work, the Developer will need to select Create Org in the Dev Org
+  card.
+- Dev Org creation can take a number of minutes. Feel free to leave the page.
+  Metecho will provide an alert when the Dev Org is ready.
+
+## Step 5 - Make Changes in Dev Org
+- **This step takes place in the new Salesforce Dev Org, not in Metecho.**
```

Comment:
```suggestion
- **Changes to a Dev Org but be made in the Salesforce UI, not in Metecho.**
```

Reply:
Is a `Dev Org` the same as a `Salesforce Dev Org`? Or does `Salesforce Dev Org` refer to a Dev Org in Salesforce?

Reply:
A "Dev Org" is the same thing as a "Salesforce Dev Org" (all Orgs in this context are Salesforce Orgs). Orgs exist in Salesforce, but we represent them in Metecho and provide a link for users to view them in the Salesforce UI (and we handle auto-login for users).

Orgs can only be modified via the Salesforce UI, not directly from Metecho. So users need to click the "View Org" link in Metecho, which will take them to the Salesforce page for that Org, where they can then make changes (e.g. adding a new field to the Org).

Reply:
> And so if the Dev Org entity is the same but users could be work on it in two places, I think it could be helpful to refer to it with the same name throughout the guide but with a location qualifier when needed like `Dev Org in Salesforce` and `Dev Org in Metecho`. What do you think @SondraE @jgerigmeyer? I think that is what I was trying to capture with the suggestion above as well.

@sanajaved7 I think that makes sense. Changes are made in the Salesforce UI, but orgs are created/deleted and changes are captured via the Metecho UI.